src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -290,7 +290,7 @@ void MulticopterPositionControl::Run()
 
 		if (_vehicle_control_mode.flag_multicopter_position_control_enabled) {
 
-			_trajectory_setpoint_sub.update(&_setpoint);
+			const bool is_traj_sp_updated = _trajectory_setpoint_sub.update(&_setpoint);
```

Comment:
```suggestion
			const bool is_trajectory_setpoint_updated = _trajectory_setpoint_sub.update(&_setpoint);
```
I would have written it out but it's an unimportant detail.